.github/ISSUE_TEMPLATE/Support_Question.md

```diff
@@ -0,0 +1,22 @@
+---
+name: ❓ Support Question
+about: Have a problem that you can't figure out? 🤔
+---
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q            |   A
+|---------------- | ------
+| Library version | x.y.z <!-- provide patch version too -->
+| PHP version     | x.y.z
```

Comment:
does it make sense to indicate this for this type of product?

Reply:
I supposed that it could be useful but if you think that PHP version is not necessary I trust in you

Reply:
it may be useless, but ok, let's try to leave it and watch how it sounds!